src/components/SelectionList/Search/TransactionGroupListItem.tsx

```diff
@@ -170,8 +170,13 @@ function TransactionGroupListItem<TItem extends ListItem>({
 
     useSyncFocus(pressableRef, !!isFocused, shouldSyncFocus);
 
+    const pendingAction =
+        (item.pendingAction ?? groupItem.transactions.every((transaction) => transaction.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE))
```

Comment:
We should have checked if we have transactions in the first place. If we don't, this will return true even though we don't have any pending action.